recipes/devices/families/kvims.sh

```diff
@@ -7,6 +7,7 @@
 BASE="Debian"
 ARCH="armhf"
 BUILD="armv7"
+UINITRD_ARCH="arm"
```

Comment:
This should be arm64

Reply:
I wonder if it is backwards compatible and so (`arm` uInitrd) will be read by an `arm64` processor, making this entire exercise a bit futile?

Reply:
could it also be related to the u-boot version? Let’s use the arch we had in the old build just now, but also look into u-boot as this did have issues in some cases